styles.css

```diff
@@ -266,26 +271,31 @@ header {
   height: 80px;
 }
 
+.bullet {
+  width: 8px;
+  height: 8px;
+  border-radius: 50%;
+  background: #c1c7d0;
+}
+
 .Primary-Text h2 {
   padding: 0 16px;
   width: 295px;
   height: 36px;
   font-family: 'Poppins', sans-serif;
   font-style: normal;
   font-weight: 700;
-
-  /* font-size: 32px; */
   line-height: 44px;
   color: #172b4d;
 }
 
 .Primary-Text ul {
-  padding: 16px 16px;
+  padding-left: 16px;
   list-style-type: none;
   display: flex;
   flex-direction: row;
   align-items: center;
-  gap: 12px;
+  gap: 7px;
 
   /* width: 236px; */
```

Comment:
- [ ] KIndly remove all commented code from your codebase so that your codebase can be clean and readable 😄